gulpfile.js

```diff
@@ -120,6 +121,12 @@ gulp.task("compile:favicons", function() {
       .pipe(gulp.dest("dist/favicons"));
 });
 
+gulp.task('validate:example', 'validate example html files', function() {
+  return gulp.src('src/*.html')
+    .pipe(compileExample('./src/templates/', 'example.html'))
```

Comment:
We could avoid duplicating the compileExample step by adding a dependency to the build (and clean) task. For a sample see the deploy task, which does the same.